src/bin/commands/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod startup;
```

Comment:
As mentioned I think it would be better one of these two options:
1. Remove the startup module and include the function as private ones directly in the `main.rs` of the binary

2. Add the startup function in the library instead, so it could be used by both main.rs and integration tests.

If you chose for option 2 I would prefer to avoid `rusk::startup::startup` and having just `rusk::startup` or something like (so re-exporting).

Reply:
Moved to to `main.rs` since it's the easiest thing to do. We won't do any fancy stuff on the tests now. Maybe we can replace this on the future.